packages/aws-otel-collector/aws-otel-collector-tmpfiles.conf

```diff
@@ -0,0 +1 @@
+C /etc/aws-otel-collector.yaml - - - - -
```

Comment:
Don't you have to create the `/var/log/aws` directory, or is something else creating it?

Reply:
The collector creates it [here](https://github.com/aws-observability/aws-otel-collector/blob/main/pkg/logger/logger.go#L82), so we don't have to.